numpy/doc/broadcasting.py

```diff
@@ -62,7 +62,10 @@
 ``ValueError: operands could not be broadcast together`` exception is 
 thrown, indicating that the arrays have incompatible shapes. The size of 
 the resulting array is the maximum size along each dimension of the input 
```

Comment:
I'd prefer a more generalizable phrasing like _"is the size that is not 1 the along each axis of the inputs"